src/test/java/guitests/ChromeDriverTest.java

```diff
@@ -32,138 +40,118 @@ public void launchApp() {
     }
 
     @Test
-    public void chromeDriverStubTest() {
-        clearUrl();
-        clearScript();
-        clearKeyCode();
-
-        UI.events.registerEvent((NavigateToPageEventHandler) e -> url = e.url);
-        UI.events.registerEvent((ExecuteScriptEventHandler) e -> script = e.script);
-        UI.events.registerEvent((SendKeysToBrowserEventHandler) e -> keyCode = e.keyCode);
-        UI.events.registerEvent((JumpToCommentEventHandler) e -> jumpToComment = true);
+    public void navigateToPageTest() {
+        UI.events.registerEvent((NavigateToPageEventHandler) e -> urlsNavigated.add(e.url));
 
         UI.events.triggerEvent(new IssueSelectedEvent("dummy/dummy", 1, 0, false));
-        sleep(EVENT_DELAY);
-        assertEquals(GitHubURL.getPathForIssue("dummy/dummy", 1), url);
-        clearUrl();
+        awaitCondition(() -> urlsNavigated.contains(GitHubURL.getPathForIssue("dummy/dummy", 1)));
+        clearUrlsNavigated();
         UI.events.triggerEvent(new IssueCreatedEvent());
-        sleep(EVENT_DELAY);
-        assertEquals(GitHubURL.getPathForNewIssue("dummy/dummy"), url);
-        clearUrl();
+        awaitCondition(() -> urlsNavigated.contains(GitHubURL.getPathForNewIssue("dummy/dummy")));
+        clearUrlsNavigated();
         UI.events.triggerEvent(new LabelCreatedEvent());
-        sleep(EVENT_DELAY);
-        assertEquals(GitHubURL.getPathForNewLabel("dummy/dummy"), url);
-        clearUrl();
+        awaitCondition(() -> urlsNavigated.contains(GitHubURL.getPathForNewLabel("dummy/dummy")));
+        clearUrlsNavigated();
         UI.events.triggerEvent(new MilestoneCreatedEvent());
-        sleep(EVENT_DELAY);
-        assertEquals(GitHubURL.getPathForNewMilestone("dummy/dummy"), url);
-        clearUrl();
-
-        clickIssue(0, 9);
-        sleep(EVENT_DELAY);
-        clearUrl();
+        awaitCondition(() -> urlsNavigated.contains(GitHubURL.getPathForNewMilestone("dummy/dummy")));
```

Comment:
This line seems to repeat a few times. Any chance it can be abstracted into a utility method?